components/script/dom/node.rs

```diff
@@ -2426,6 +2417,21 @@ impl Node {
                 .map_or(ns!(), |elem| elem.locate_namespace(prefix)),
         }
     }
+
+    /// If the given untrusted node address represents a valid DOM node in the given runtime,
+    /// returns it.
+    #[allow(unsafe_code)]
+    pub unsafe fn from_untrusted_node_address(candidate: UntrustedNodeAddress) -> &'static Self {
+        // https://github.com/servo/servo/issues/6383
+        let candidate = candidate.0 as usize;
+        //        let object: *mut JSObject = jsfriendapi::bindgen::JS_GetAddressableObject(runtime,
+        //                                                                                  candidate);
```

Comment:
I guess we can remove the dead  code as well? I'm not sure what `jsfriendapi` is...

Reply:
Yeah,this code has not worked for at least 8 years.